.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: Build Splunk App
+on:
+  push:
+    branches:
+      - "master"
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+  pull_request:
+    branches:
+      - "master"
+
+
+jobs:
+
+  # Package App
+  package:
+    name: Package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # semantic-release won't trigger a tagged build if this is not set false
+          persist-credentials: false
+
+      # Our add-on contains Python code, so we need to install Python in the container
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Package Splunk App with CLI
+        run: |
+          pwd
+          ls -la
+          cd ${GITHUB_WORKSPACE}
+          mkdir dist
+          tar -zcvf dist/sekoia.io-current.spl sekoia.io
+
+      - name: Upload package as an artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: package-splunkbase
+          path: dist/sekoia.io-current.spl
+        if: always()
```

Comment:
why ? Wouln't it fail if previous steps were unsuccessful ?

Reply:
Well spotted, I removed it.